assets/src/styles/blocks/CarouselHeader/CarouselHeaderStyle.scss

```diff
@@ -535,6 +540,15 @@ $medium-image-height: 600px;
         background-color: rgba(0, 0, 0, .15);
         transform: translateX(-5px);
       }
+
+      html[dir="rtl"] & {
+        right: 0;
+        left: auto;
+
+        &:hover {
+          transform: translateX(5px);
```

Comment:
Does it have to be translate? Perhaps adjusting margin/padding would work too?

Reply:
And it actually also makes sense here :ok_hand:

Reply:
Updated to use padding instead 🙂

Reply:
Just to document it here as well: we noticed that using `translateX` results in the button moving from under the cursor, if the cursor is within 5px of the edge. Because there is also a transition speed this results in the button moving back and forth a few times (exact behavior probably varies across browsers/screen sizes). Though quickly and not in all cases, so a very minor issue.

Using padding both fixes that and avoids needing to change the value for RTL. With `translateX` we would need to override this with the opposite value for RTL, for padding it works in both cases.